### PR TITLE
add script for dodrio quota

### PR DESCRIPTION
**Pull request by kwaegema**

### CODE REVIEW

File: bin/my_dodrio_quota.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: latin-1 -*-
```
Comment (wdpypere) [inline bin/my_dodrio_quota.py]: why is this line needed?

Comment (kwaegema) [reply bin/my_dodrio_quota.py]: don't know, as you could have guessed, i didn't write the copyright statement myself but I copied it from another script, which happened to have that line :smile:

Comment (wdpypere) [reply bin/my_dodrio_quota.py]: remove it, luis his name is not in here ;)
